yaml/com/ibm/Control/Host/PcieLink.interface.yaml

```diff
@@ -0,0 +1,10 @@
+description: >
+    Pcie Link interface is used to reset the link
+    to true or false.
```

Comment:
"Pcie Link interface is used to reset the link to true or false." doesn't make sense to me. 
     "PCIe Link Interface is used to reset the PCIe link" ?

Reply:
done